config/jobs/kubernetes/publishing-bot/publishing-bot-periodics.yml

```diff
@@ -0,0 +1,87 @@
+periodics:
+- name: periodic-publishing-bot
+  cluster: k8s-infra-prow-build-trusted
+  annotations:
+    testgrid-dashboards: sig-release-publishing-bot
+    testgrid-tab-name: periodic
+  spec:
+    initContainers:
+    - name: initialize-repos
+      command:
+      - /init-repo
+      - --alsologtostderr
+      - --config=/etc/munge-config/config
+      - --rules-file=/etc/publisher-rules/config
+      - 2>&1
+      image: gcr.io/k8s-staging-publishing-bot/k8s-publishing-bot
+      imagePullPolicy: Always
+      resources:
+        requests:
+          cpu: 300m
+          memory: 200Mi
+        limits:
+          cpu: 2
+          memory: 1639Mi
+      volumeMounts:
+      - mountPath: /etc/munge-config
+        name: munge-config
+      - mountPath: /go-workspace
+        name: publisher-gopath
+      - mountPath: /etc/publisher-rules
+        name: publisher-rules
+      - mountPath: /.cache
+        name: cache
+    containers:
+    - name: publisher
+      command:
+      - /publishing-bot
+      - --alsologtostderr
+      - --config=/etc/munge-config/config
+      - --token-file=/etc/secret-volume/token
+      - --interval=0
+      - --server-port=8080
+      - 2>&1
+      image: gcr.io/k8s-staging-publishing-bot/k8s-publishing-bot
+      imagePullPolicy: Always
+      resources:
+        requests:
+          cpu: 300m
+          memory: 200Mi
+        limits:
+          cpu: 2
+          memory: 1639Mi
```

Comment:
Can we keep the resource values in sync with the one used currently here https://github.com/kubernetes/publishing-bot/blob/master/configs/kubernetes